### ISSUE TITLE
SSD300 and SSD512 aspect ratios

### ISSUE THREAD

**Issue by i-TechSoftware** [pierluigiferrari/ssd_keras#333]
Hello!
Thanks for the opportunity to learn more about the SSD architecture, it really helped me!
I read the documentation, but I have two questions left, and I would really appreciate it if you could take the time to answer them.
I trained the SSD300 and SSD512 network on my dataset which had 2 classes (background and car license plate). Image sizes I provided for training 512x512px. After training, both networks perfectly recognize car license plates. I want to know if I can provide images for training with aspect ratios other than 1: 1 (for example 1: 2.5) without resizing during the training? How will this affect learning? Or, given the current architecture of the SSD network, only 1: 1 aspect ratios are needed?

Second question:
The graph that is attached to the message shows a strange variation in data validation. I don’t know how it is correct in English - it looks like a comb :). 
(
![val_loss](https://user-images.githubusercontent.com/48829872/73851161-12df0780-483e-11ea-9dd2-5eed23db4086.jpg)
![train_loss](https://user-images.githubusercontent.com/48829872/73851185-1ecac980-483e-11ea-9448-e8a08b8a92d9.jpg)

I have two assumptions about this:
1. I have little data (I trained on a dataset of 2000 images)
2. Due to the fact that I have only 2 classes. At the same time, one of them (background) as a percentage significantly outweighs the second class.
I used the Adam optimizer. And lowered the learning rate after the 30th and 100th epoch. At the end of training on validation data, the losses reached 0.6. (attached graphics from another training, but on the described training they were one to one like these)
And as I said above - the network copes with the recognition of car license plates.
Can you comment on this somehow or give some recommendations?

I use
python 3.6.8 (default, Oct 7 2019, 12:59:55)
[GCC 8.3.0]
tensorflow version 1.15.2
Keras version 2.3.1

**Comment by i-TechSoftware** [comment]
Any ideas?
